.github/workflows/scorecard.yml

```
@@ -0,0 +1,47 @@
+name: Scorecard supply-chain security
+
+on:
+  # Branch-Protection チェック用（デフォルトブランチのみ対応）
+  branch_protection_rule:
+  # Maintained チェックを定期的に更新
+  schedule:
+    - cron: '20 7 * * 2'
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)